File: Py (3)/1.py
def suma_vector():
    n = int(input("Ingrese la cantidad de elementos: "))
    vector = []
    suma = 0

    for i in range(n):
        valor = int(input(f"Elemento {i+1}: "))
        vector.append(valor)
        suma += valor

    print("La suma total de los elementos es:", suma)

# Ejecutar
suma_vector()

File: C++(3)/1.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int n;
    cout << "Ingrese la cantidad de elementos: ";
    cin >> n;

    vector<int> vector(n);
    int suma = 0;

    for (int i = 0; i < n; i++) {
        cout << "Elemento " << i+1 << ": ";
        cin >> vector[i];
        suma += vector[i];
    }

    cout << "La suma total de los elementos es: " << suma << endl;
    return 0;
}